client/src/components/new-request/analyze-results.vue

```diff
@@ -78,17 +79,19 @@
                 </v-col>
               </v-row>
 
-            <!--GREY BOXES-->
-            <v-row class="row colour-p-blue-text no-gutters justify-center">
-              <v-col :key="issue.issue_type + '-' + option.header + '-' + optionIndex"
-                     cols="auto"
-                     v-for="(option, optionIndex) of issue.setup">
-                <GreyBox :i="optionIndex"
-                         :issueIndex="issueIndex"
-                         :option="option"
-                         :originalName="originalName" />
-              </v-col>
-            </v-row>
+              <!--GREY BOXES-->
+              <v-row class="colour-p-blue-text justify-center" dense v-if="showGreyBoxes">
+                <v-col :key="issue.issue_type + '-' + option.header + '-' + optionIndex"
+                       v-for="(option, optionIndex) of issue.setup">
+                  <GreyBox :changesInBaseName="changesInBaseName"
+                           :designationIsFixed="designationIsFixed"
+                           :i="optionIndex"
+                           :issueIndex="issueIndex"
+                           :option="option"
+                           :originalName="originalName"
+                           class="issue.setup.length === 3 && i === 1 ? 'mx-2' : ''" />
```

Comment:
This won't work until you change it to `:class`.
(I've made this mistake before :P )

Reply:
@scottrumsby Does this class property actually work?

Reply:
its works in that it doesn't do anything.  will delete it.